OWNERS.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
this file needs to have maintainers listed with their contact info, e.g. https://github.com/crossplane-contrib/provider-upjet-aws/blob/main/OWNERS.md